api/src/pcapi/connectors/boost.py

```diff
@@ -9,14 +9,31 @@
 from pcapi import settings
 from pcapi.connectors.serialization.boost_serializers import LoginBoost
 from pcapi.core.external_bookings.boost.exceptions import BoostAPIException
+from pcapi.core.external_bookings.boost.exceptions import BoostInvalidTokenException
+from pcapi.core.external_bookings.boost.exceptions import BoostLoginException
 from pcapi.core.providers.models import BoostCinemaDetails
 import pcapi.core.providers.repository as providers_repository
 from pcapi.repository import repository
 from pcapi.routes.serialization import BaseModel
 from pcapi.utils import requests
+from pcapi.utils.decorators import retry
 
 
 logger = logging.getLogger(__name__)
+ATTEMPTS_LIMIT = 2
+
+
+def invalid_token_handler(
+    resource: "ResourceBoost",
+    cinema_details: BoostCinemaDetails,
+    pattern_values: dict[str, Any] | None = None,
+    params: dict[str, Any] | None = None,
```

Comment:
(détail) Puisqu'on n'utilise pas ces deux arguments, on pourrait simplifier leur typage : 

```suggestion
    pattern_values: dict | None = None,
    params: dict | None = None,
```

Reply:
done @dbaty